core/src/subgraph/instance.rs

```diff
@@ -28,6 +28,13 @@ where
         // event processing behavior predictable
         let manifest_id = manifest.id;
         let spec_version = Version::parse(&manifest.spec_version)?;
+        if let false = VersionReq::parse("<= 0.0.2")?.matches(&spec_version) {
```

Comment:
You can unwrap this, it will never fail. Also `if let false = expr` is just `if !expr` 😄